planning/behavior_velocity_planner/src/scene_module/detection_area/scene.cpp

```diff
@@ -223,19 +221,44 @@ bool DetectionAreaModule::modifyPathVelocity(
     last_obstacle_found_time_ = std::make_shared<const rclcpp::Time>(clock_->now());
   }
 
+  // Get stop line geometry
+  const auto stop_line = getStopLineGeometry2d();
+
+  // Get self pose
+  const auto & self_pose = planner_data_->current_pose.pose;
+
+  // Get stop point
+  const auto stop_point = createTargetPoint(original_path, stop_line, planner_param_.stop_margin);
+  if (!stop_point) {
+    return true;
+  }
+
+  const auto is_stopped = planner_data_->isVehicleStopped(0.0);
+
+  auto stop_pose = stop_point->second;
+  const auto stop_dist = calcSignedArcLength(path->points, self_pose.position, stop_pose.position);
+
+  // Don't re-approach when the ego stops closer to the stop point than hold_stop_margin_distance
+  if (is_stopped && stop_dist < planner_param_.hold_stop_margin_distance) {
+    const auto ego_pos_on_path =
+      calcLongitudinalOffsetPose(original_path.points, self_pose.position, 0.0);
+
+    if (!ego_pos_on_path) {
+      return false;
+    }
+
+    stop_pose = ego_pos_on_path.get();
+  }
+
+  setDistance(stop_dist);
```

Comment:
setDistance関数は[RTC](https://github.com/autowarefoundation/autoware.universe/pull/963/files)のためのものであるため、v0.3.xとしては対象外であるため削除。
```suggestion

  setDistance(stop_dist);
```